src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.1.14
```

Comment:
You don't need to bump the version. These changes can be kept under 0.1.13. The latest released version is https://pypi.python.org/pypi/azure-cli-container/0.1.12